src/optimization-given.jl

```diff
@@ -92,11 +92,11 @@ function optimization_given(allowgw=false, allowreservoirs=true, demandmodel=not
         #cwro = deserialize(open(cachepath("partialhouse2$suffix.jld"), "r"));
         cwro = constraintoffset_waternetwork_outflows(m);
```

Comment:
This should actually refer to the EnvironmentalFlows component, say be subtracting it from the basic constraint offset, in the final version.

Reply:
Indeed. This is now at line 56 in EnvironmentalDemand.jl
I was a bit reluctant as it means adding and initiating an environmentaldemand component for not much, as it takes the same form as in WaterNetwork.jl - but it is indeed less messy.